functions/src/definitions/common/utils.ts

```diff
@@ -33,6 +33,13 @@ const checkUrl = function (urlString: string) {
   return url.protocol === "http:" || url.protocol === "https:"
 }
 
+async function checkMessageId(messageId: string) {
+  const db = admin.firestore()
+  const messageRef = db.doc(`messagesIds/${messageId}`)
```

Comment:
Not sure if this is the correct syntax as the samples in firebase docs seems to require creating a similar to `const messageIdRef = db.collection("messageIds").doc(id)` found below.

Also, `message(s)Ids` vs `messageId` looks suspicious

Reply:
Good catch, fixed!